ui/flake-module.nix

```
@@ -0,0 +1,83 @@
+{ self, ... }:
+{
+  perSystem = { config, self', inputs', system, ... }:
+    let
+      pkgs = inputs'.nixpkgs.legacyPackages;
+      inherit (config) cat-lib dream2nix;
+      ui =
+        (dream2nix.lib.makeOutputs { source = ./.; settings = [{ subsystemInfo.nodejs = 16; }]; }).packages.danaswap-ui;
+    in
+    {
+      packages = {
+        danaswap-ui =
+          pkgs.runCommand "build-danaswap-ui"
+            { buildInputs = [ self'.packages."offchain:danaswap-browser" ]; }
```

```suggestion
            { }
```
not needed as you directly refer to the package using interpolation below